media/css/mozorg/home/home-new.scss

```diff
@@ -188,30 +188,17 @@ header {
     }
 
     .c-foundation {
-        a {
+        a.mzp-c-button[rel~="external"] {
             display: inline-flex;
             align-items: center;
 
```

Comment:
**Nit + suggestion:** The icon is shrinking to smaller sizes when the window is smaller. Adding a `flex-shrink: 0` to the `span` element holding the `svg` should fix this!

<kbd>![Screenshot 2024-04-23 at 4 23 56 PM](https://github.com/mozilla/bedrock/assets/42309026/c70f92d1-dc09-4c6d-a71f-cccbdb6e6393)</kbd>

Reply:
You're absolutely right, fixed.🤦 How did I miss this? Thanks!! Really appreciate you triple quadruple checking even the most trivial changes after me turning knobs and touching buttons to see what explodes where;D

I originally wanted to get rid of the inline-flex altogether, but the way the base `button-icon-end` leaves the icon "dangling" on an empty new line, inlined/treated like text for longer CTAs like this, doesn't look very appealing. So I kept the idea of the original author, to improve the relation between text and icon, and it's way better.

I'm actually trying this button as a proof-of-concept for https://github.com/mozilla/bedrock/pull/14407#pullrequestreview-2006649547 that could benefit from it too, with so many longer CTAs suffering from the orphaned icons on new lines.